Engine/source/console/simFieldDictionary.cpp

```diff
@@ -168,6 +168,25 @@ SimFieldDictionary::Entry *SimFieldDictionary::findDynamicField( StringTableEntr
    return NULL;
 }
 
+void SimFieldDictionary::clearFields()
+{
+	for(U32 i = 0; i < HashTableSize; i++)
+   {
+		Entry **walk = &mHashTable[i];
+		Entry *field = NULL;
+		while(*walk)
+		{
+			field = *walk;
+			mVersion++;
+
+			if( field->value )
+				dFree(field->value);
```

Comment:
I would set field->value to NULL too otherwise the entry will be placed back into the smFreeList with an invalid pointer.